docs/docsite/rst/network/user_guide/platform_eos.rst

```diff
@@ -0,0 +1,129 @@
+.. _eos_platform_options:
+
+***************************************
+EOS Platform Options
+***************************************
+
+Arista EOS supports multiple connections. This page offers details on how each connection works in Ansible 2.5 and how to use it. 
+
+.. contents:: Topics
+
+Connections Available
+================================================================================
+
++---------------------------+-----------------------------------------------+-----------------------------------------+
+|..                         | CLI                                           | eAPI                                    |
++===========================+===============================================+=========================================+
+| **Protocol**              |  SSH                                          | HTTP(S)                                 |
++---------------------------+-----------------------------------------------+-----------------------------------------+
+| | **Credentials**         | | uses SSH keys / SSH-agent if present        | | uses HTTPS certificates if present    |
+| |                         | | accepts ``-u myuser -k`` if using password  | |                                       |
++---------------------------+-----------------------------------------------+-----------------------------------------+
+| **Indirect Access**       | via a bastion (jump host)                     | via a web proxy                         |
++---------------------------+-----------------------------------------------+-----------------------------------------+
+| | **Connection Settings** | | ``ansible_connection: network_cli``         | | ``ansible_connection: local``         |
+| |                         | |                                             | | Requires ``transport: eapi``          |
+| |                         | |                                             | | in the ``provider`` dictionary        |
++---------------------------+-----------------------------------------------+-----------------------------------------+
+| | **Enable Mode**         | | supported - use ``ansible_become: yes``     | | supported - use ``authorize: yes``    |
+| | (Privilege Escalation)  | | with ``ansible_become_method: enable``      | | in the ``provider`` dictionary        |
```

Comment:
Mention about `ansible_become_pass` and `auth_pass` to make the connection parm difference more clear?

Reply:
Thanks for the feedback. Incorporated in d0d7d916162a3f8b6e5a6b24b2d2ab715124526a